android/java/org/chromium/chrome/browser/crypto_wallet/util/Utils.java

```diff
@@ -996,6 +1016,10 @@ public static BlockchainToken makeNetworkAsset(NetworkInfo network) {
         } else if (network.chainId.equals(
                            BraveWalletConstants.BINANCE_SMART_CHAIN_MAINNET_CHAIN_ID)) {
             logo = "bnb.png";
+        } else if (network.chainId.equals(BraveWalletConstants.SOLANA_MAINNET)
+                || network.chainId.equals(BraveWalletConstants.SOLANA_TESTNET)
+                || network.chainId.equals(BraveWalletConstants.SOLANA_DEVNET)) {
+            logo = "sol.png";
```

Comment:
@SergeyZhukovsky do we have to hard code file names here ?

Reply:
that's how we do for now. For eth.png case we download it via components and check using that name, for sol.png we don't have it yet in components, but perhaps will